src/components/ColorTokenTable/color-token-table.scss

```diff
@@ -6,9 +6,18 @@
 .color-token-table__theme-switcher {
   position: relative;
   width: 100%;
-  height: rem(48px);
+  height: 3rem;
   margin-bottom: 1.75rem;
   background: $ui-02;
+
+  @media all and (max-width: 1055px) {
```

Comment:
Quick question, where are the breakpoint values coming from? Are they tied to a range of device sizes or a particular layout shift that's happening?

Reply:
Hmmmm, I'm not sure. This was an existing style that I just moved up.

Reply:
It appears that it is part of our breakpoints (Large = 1056px).

Reply:
I'll update it to use our mixin/breakpoint token